src/cargo-fmt/main.rs

```diff
@@ -598,13 +591,15 @@ mod cargo_fmt_tests {
             "p1",
             "-p",
             "p2",
+            "--offline",
             "--message-format",
             "short",
             "--",
             "--edition",
             "2018",
         ]);
         assert_eq!(true, o.quiet);
+        assert_eq!(true, o.offline);
```

Comment:
I think that unit test is good enough... @topecongiro ?

Reply:
It'd be cool if one day we could have tests that validate that the correct options were passed to the `cargo_metadata` lib based on command line items passed to `cargo fmt`, though that seems like it might be a bit tricky